plugins/cmp/auto-enable.nix

```diff
@@ -69,6 +69,11 @@ in
           If you want to keep using raw lua for defining your sources:
           - Ensure you enable the relevant plugins manually in your configuration;
           - Dismiss this warning by explicitly setting `autoEnableSources` to `false`;
+        ''
+        # TODO: Added 2024-09-22; remove after 24.11
+        ++ optional (elem "otter" enabledSources) ''
+          Nixvim (plugins.cmp): "otter" is listed in `settings.sources`, however it is no longer a cmp source.
+          Instead, you should enable `plugins.otter` and use the "cmp-nvim-lsp" completion source.
```

Comment:
Currently, this warning will show if `autoEnableSources` is enabled _and_ `otter` is listed in `settings.sources`.

Perhaps it would make sense for this to also be shown even when `autoEnableSources` isn't true? But that'd be a more complex change.

Reply:
> Perhaps it would make sense for this to also be shown even when autoEnableSources isn't true?

IMHO, yes. But:
1. It requires more work - per your words.
2. IMHO, very few users have `autoEnableSources = false`.

Therefor, I think what you did is enough.